implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -79,12 +79,18 @@
     private JWTAuthContextInfo authContextInfo;
 
     public KeyLocationResolver(JWTAuthContextInfo authContextInfo) throws UnresolvableKeyException {
+        if (authContextInfo == null) {
```

Comment:
This can not happen, so can be dropped IMHO. And if NPE did happen later then it would indicate a really broken default implementation :-)

Reply:
Removed the check.